src/batch-client/types.ts

```diff
@@ -146,6 +147,11 @@ export interface RelatedContactsOptions {
 	email: string;
 }
 
+export interface SaveDocumentInput {
+	requestId?: Number
```

Comment:
we should call it uploadId

Reply:
isn't this part of DocumentData ?

Reply:
it is part of Document but will be inside a structure
like```
 {
folder:folderid
upload {
    id:id
   }
documentid : documentId
}
```

so thought of this